go/vt/tabletserver/sysloglogger/sysloglogger_test.go

```diff
@@ -0,0 +1,36 @@
+package sysloglogger
+
+import (
+	"log/syslog"
+	"testing"
+)
+
+type fakeWriter struct {
+	priority syslog.Priority
+	message  string
+	err      error // if non-nil, force an error to be returned
+}
+
+func (fw *fakeWriter) write(pri syslog.Priority, msg string) error {
+	if fw.err != nil {
+		return fw.err
+	}
+
+	fw.priority = pri
+	fw.message = msg
+	return nil
+}
+
+func (fw *fakeWriter) Info(msg string) error { return fw.write(syslog.LOG_INFO, msg) }
+func (fw *fakeWriter) Close() error          { return }
+
+// TestSyslog checks that our callback works.
+func TestSyslog(t *testing.T) {
+	writer = &fakeWriter{}
+
+	// Somehow trigger some StatsLogger activity, and check the contents of "writer.message" afterward
+
+	// However, that's probably not going to be possible in a unit test context.  Perhaps refactor
+	// that call to "StatsLogger.Subscribe(...)", so that the returned channel can be replaced by some
+	// sort of mock instead?
+}
```

Comment:
I may be on the wrong path altogether here.  If you have some input on how to make unit testing possible, then I'll happily apply them.